src/DynamoPublish/ViewModels/PublishViewModel.cs

```diff
@@ -163,6 +172,33 @@ private void OnPublish(object obj)
             model.SendAsynchronously(Workspaces, workspaceProperties);
         }
 
+        private void Visit(object _)
+        {
+            try
+            {
+                System.Diagnostics.Process.Start(ShareLink);
+            }
+            catch (Exception e)
+            {
+                model.AsLogger().Log(e);
+            }
```

Comment:
Starting a process on the local machine is always a dubious process - I surround it with a try catch.